ocp_cleanup.sh

```diff
@@ -4,8 +4,10 @@ set -e
 
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster || true
-
-rm -rf ocp
-
-sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf
+if "$GOPATH/src/github.com/openshift/installer/bin/openshift-install" --log-level=debug --dir ocp destroy cluster; then
```

Comment:
For this command to succeed for me I need to add a sudo, it can then delete the bootstrap node, with an error message about storage pools
FATAL Failed to destroy cluster: get storage pool "default": virError(Code=49, Domain=18, Message='Storage pool not found: no storage pool with matching name 'default'')